lib/measured/rails/validations.rb

```diff
@@ -9,10 +9,13 @@ class MeasuredValidator < ActiveModel::EachValidator
 
   def validate_each(record, attribute, measurable)
     measured_config = record.class.measured_fields[attribute]
+    unit_field_name = measured_config[:unit_field_name] || "#{ attribute }_unit"
+    value_field_name = measured_config[:unit_field_name] || "#{ attribute }_value"
```

Comment:
👀 You're assigning `unit` to `value` here.

Also, if this didn't fail tests, we have incomplete test coverage.